modules/machine-api-operator.adoc

```diff
@@ -1,15 +1,52 @@
 // Module included in the following assemblies:
 //
 // * operators/operator-reference.adoc
+// * installing/cluster-capabilities.adoc
 
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
 [id="machine-api-operator_{context}"]
-= Machine API Operator
+ifdef::operator-ref[= Machine API Operator]
+ifdef::cluster-caps[= Machine API capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Machine API Operator is an optional cluster capability that can be disabled by cluster administrators during installation. The Machine API capability is responsible for all configuration and management of machines in the cluster. If you disable the Machine API Operator at installation, you need to manually manage all nodes and machine-related tasks. For more information about optional cluster capabilities, see "Cluster capabilities" in _Installing_.
```

Comment:
Capability MachineAPI includes three operators that can be disabled.
machine-api-operator
cluster-autoscaler-operator
cluster-control-plane-machine-set-operator